doc/api/process.md

```diff
@@ -941,12 +1208,24 @@ that exports a `foo` function. All the symbols are loaded before
 the call returns, by passing the `RTLD_NOW` constant. In this example
 the constant is assumed to be available.
 
-```js
-const os = require('os');
-const path = require('path');
+```mjs
+import { dlopen } from 'process';
+import { constants } from 'os';
+import { fileURLToPath } from 'url';
+
+const module = { exports: {} };
+dlopen(module, fileURLToPath(new URL('local.node', import.meta.url)),
+       constants.dlopen.RTLD_NOW);
+module.exports.foo();
+```
+
+```cjs
+const { dlopen } = require('process');
+const { constants } = require('os');
+const { join } = require('path');
+
 const module = { exports: {} };
-process.dlopen(module, path.join(__dirname, 'local.node'),
-               os.constants.dlopen.RTLD_NOW);
+dlopen(module, join(__dirname, 'local.node'), constants.dlopen.RTLD_NOW);
```

Comment:
I subjectively prefer `path.join` rather than simply `join`, feel free to disregard if you disagree.
```suggestion
dlopen(module, path.join(__dirname, 'local.node'), constants.dlopen.RTLD_NOW);
```

Reply:
I did go quite agressive on the destructured imports, down to leaving out cases like "on". I understand some of these get tricky, but want to try and encourage this as much as possible for tree-shaking style optimizations. In theory leaving out the member expression access is more performant as well although it's a micro-optimization for sure. Will leave this in unless anyone wants to push back further.